.github/workflows/offlinedocs.yaml

```diff
@@ -0,0 +1,77 @@
+name: offlinedocs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "offlinedocs/**"
+  pull_request:
+    paths:
+      - "offlinedocs/**"
+  workflow_dispatch:
+
+# Cancel in-progress runs for pull requests when developers push
+# additional changes
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  fmt:
+    name: fmt
+    runs-on: ${{ github.repository_owner == 'coder' && 'buildjet-8vcpu-ubuntu-2204' || 'ubuntu-latest' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Pretier
+        run: |
+          cd offlinedocs
+          mv package.json package.json.bak
+          jq '{dependencies: {prettier: .devDependencies.prettier}}' < package.json.bak > package.json
+          yarn --frozen-lockfile
+          mv package.json.bak package.json
+
+      - name: Check format
+        run: |
+          cd offlinedocs
+          yarn format:check
+
+  lint:
+    name: lint
+    runs-on: ${{ github.repository_owner == 'coder' && 'buildjet-8vcpu-ubuntu-2204' || 'ubuntu-latest' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: buildjet/setup-node@v3
+        with:
+          node-version: 16.20.1
+          cache: "yarn"
+          cache-dependency-path: "offlinedocs/yarn.lock"
```

Comment:
This should use our setup-node action. You could add a `dir` input to the action (which defaults to `site`) and change it to `offlinedocs` for this.